From: Juergen Beisert <jbeisert@pengutronix.de>
Date: Sun, 30 Oct 2011 22:33:40 +0100
Subject: [PATCH] fix a simple compile error

This fixes the following compile error:

.libs/drivers.o drivers.c:938: error: expected '}' before ';' token

Signed-off-by: Juergen Beisert <jbeisert@pengutronix.de>
---
# 20110222 wsa: fixed in master meanwhile

 drivers.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers.c b/drivers.c
index 89cd772..373476f 100644
--- a/drivers.c
+++ b/drivers.c
@@ -935,7 +935,7 @@ static gps_mask_t garmintxt_parse_input(struct gps_device_t *session)
 
 static const struct gps_type_t garmintxt = {
     .type_name     = "Garmin Simple Text",		/* full name of type */
-    .packet_type   = RTCM2_PACKET;	/* associated lexer packet type */
+    .packet_type   = RTCM2_PACKET,	/* associated lexer packet type */
     .trigger       = NULL,		/* no recognition string */
     .channels      = 0,			/* not used */
     .probe_wakeup  = NULL,		/* no wakeup to be done before hunt */
